Gemfile

```diff
@@ -1,19 +1,126 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-group :development, :test do
-  gem 'rake',                    :require => false
-  gem 'rspec-puppet',            :require => false
-  gem 'puppetlabs_spec_helper',  :require => false
-  gem 'beaker-rspec',            :require => false
-  gem 'puppet-lint',             :require => false
-  gem 'pry',                     :require => false
```

Comment:
I think we lost pry, but that's probably okay.

Reply:
Nothing a .sync.yml can't fix.